smalltalksrc/Slang-Tests/SLBasicTestDeclarationClass.class.st

```diff
@@ -4,6 +4,12 @@ Class {
 	#category : #'Slang-Tests'
 }
 
+{ #category : #translation }
+SLBasicTestDeclarationClass class >> typeForSelf [
+
+	^#implicit
+]
```

Comment:
The inherited behaviour from `SlangClass` is to return `nil`. Or are you saying that at some point that `nil` defaults to `#implicit`?

Reply:
I was just asking for understanding, because this definition is in the tests, but maybe could be in a more general place? I don't know.
Anyway, it is out of the scope of this PR, so we can continue with our life.